src/frontend/magnify/apps/magnify-site/src/views/rooms/settings/index.test.tsx

```diff
@@ -26,7 +26,33 @@ describe('RoomSettingsView', () => {
       { initialEntries: ['/rooms/123/settings'], initialIndex: 1 },
     );
 
-    render(<TestingContainer router={router} />);
-    await screen.findByText('Room settings');
+    await render(<TestingContainer router={router} />);
+    await screen.findByText('Settings');
+    screen.getByText('Room settings');
+    screen.getByText('Members');
+  });
+
+  it('renders privfate room', async () => {
+    server.use(
+      rest.get(buildApiUrl('/rooms/:id/'), (req, res, ctx) => {
+        const room = createRandomRoom();
+        room.is_public = true;
+        return res(ctx.json(room));
+      }),
+    );
+    const router = createMemoryRouter(
+      [
+        {
+          path: '/rooms/:id/settings',
+          element: <RoomSettingsView />,
+        },
+      ],
+      { initialEntries: ['/rooms/1234/settings'], initialIndex: 1 },
+    );
+
+    await render(<TestingContainer router={router} />);
+    await screen.findByText('Settings');
+    screen.getByText('Room settings');
+    expect(screen.queryByText('Members')).toBe(null);
```

Comment:
If the room is public, we should expect that RoomUsersConfig has not been rendered.

Reply:
Yes, this is the point for this test, can you explain ?